AnkiDroid/src/main/java/com/ichi2/anki/ui/windows/reviewer/AnswerButton.kt

```diff
@@ -24,41 +24,61 @@ import com.google.android.material.button.MaterialButton
 import com.ichi2.anki.R
 import com.ichi2.anki.utils.ext.usingStyledAttributes
 
-class AnswerButton
-    @JvmOverloads
+class AnswerButton : MaterialButton {
+    private val easeName: String
+    constructor(context: Context) : this(context, null)
     constructor(
         context: Context,
-        attrs: AttributeSet? = null,
-        defStyleAttr: Int = com.google.android.material.R.attr.materialButtonStyle,
-    ) : MaterialButton(context, attrs, defStyleAttr) {
-        private val easeName: String =
+        attrs: AttributeSet?,
+    ) : this(context, attrs, com.google.android.material.R.attr.materialButtonStyle)
+    constructor(
+        context: Context,
+        attrs: AttributeSet?,
+        defStyleAttr: Int,
+    ) : super(context, attrs, defStyleAttr) {
+        easeName =
             context.usingStyledAttributes(attrs, R.styleable.AnswerButton) {
                 requireNotNull(getString(R.styleable.AnswerButton_easeName)) {
                     "app:easeName value not set"
                 }
             }
+        val nextTime =
+            context.usingStyledAttributes(attrs, R.styleable.AnswerButton) {
+                getString(R.styleable.AnswerButton_nextTime)
+            }
+        setNextTime(nextTime)
+    }
 
-        init {
-            val nextTime =
-                context.usingStyledAttributes(attrs, R.styleable.AnswerButton) {
-                    getString(R.styleable.AnswerButton_nextTime)
+    fun setNextTime(nextTime: String?) {
+        text =
+            if (nextTime != null) {
+                buildSpannedString {
+                    inSpans(RelativeSizeSpan(0.8F)) {
+                        append(nextTime)
+                    }
+                    append("\n")
+                    append(easeName)
                 }
+            } else {
+                easeName
+            }
```

Comment:
totally optional, readability

```suggestion
    text = buildSpannedString {
        if (nextTime != null) {
            inSpans(RelativeSizeSpan(0.8f)) { append(nextTime) }
            append("\n")
        }
        append(easeName)
    }
```